client/src/Controller/UserController.php

```diff
@@ -368,7 +366,31 @@ public function agreeTermsUseAction(Request $request, string $token): Response
     }
 
     /**
-     * @param EntityDir\User $user
+     * @Route("/user/update-terms-use/{token}", name="user_updated_terms_use")
```

Comment:
As this is intended purely for org users may be worth adding an `is_granted()` annotation here to make it clear for future users. If you check in the client `securty.yml` it looks like a `ROLE_ORG` should be sufficient as it's at the top of the hierarchy for PA and PROF roles.